package.json

```diff
@@ -25,6 +25,7 @@
     "babel-loader": "7.1.1",
     "babel-plugin-add-module-exports": "0.2.1",
     "babel-plugin-syntax-jsx": "6.18.0",
+    "babel-plugin-transform-builtin-extend": "1.1.2",
```

Comment:
I’m wondering if it can be moved to dev dependencies if it would be used only for test env.

Reply:
That's a good suggestion. I've moved it into dev dependencies and only enable it in test env.